### DISCUSSION TITLE
Python noob but upcoming BM here who needs help

### DISCUSSION THREAD

**Discussion by SirAzuu** [qu-gg/BnB-LootGenerator#28] [General]
Dear authors. 

As the title suggests, I'm a newcomer to programming in general but I am really looking forward to using the code to generate guns and loot in an upcoming BnB campaign. I have been trying out a lot of things to get the script running but I keep getting errors in PyCharm when it comes to installing some dependencies like fitz, numpys, panda and pillow. 
What do I do wrong?

Thanks for the help!
Cheers

**Comment by qu-gg** [comment]
Hi there! Thanks for the interest in the program!

If you're only interested in using the program and not modifying any of the actual code, then you can grab the latest release from the Releases tab: https://github.com/qu-gg/BnB-LootGenerator/releases/tag/v1.5.0. There just download the .zip, extract, and run the .exe to launch the UI. If you want to also add new items and such, then you can just add new entries to the JSON files in the resources folders - a tutorial is available in the README.

If you're interested in actually modifying the source code (adding new UI features, functionality, etc), then I'd be happy to help sort out dependencies. I'd just need to see pictures of the specific errors you're getting there.

Thanks!
Quagg
